choreolib/src/main/java/choreo/trajectory/ChorTrajectory.java

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Choreo contributors
+
+package choreo.trajectory;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/** A trajectory loaded from Choreo. */
+public class ChorTrajectory<SampleType extends TrajSample<SampleType>> {
```

Comment:
It being `Trajectory` was annoying because the intellisense would always put the wpilib trajectory above the choreo one so I would have to arrow key down twice to auto complete it.

Reply:
Alphabetical sorting over other things is a bad reason to choose the longer name.

Reply:
Its not for alphabetical, its so you start typing it differently, starting with `Chor` will always pull it up. I agree with banks that its important for this one class. This class will also likely only be used by most users in one place, their trajectory logger lambda.

Reply:
What's the point of even using packages if they don't disambiguate anything?